catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -48,10 +48,11 @@
             description="Enables versioning of both metacards and content."/>
     </OCD>
 
-    <OCD name="Source Poller" id="ddf.catalog.util.impl.SourcePoller">
-        <AD name="Interval" id="interval" type="Integer"
+    <OCD name="Source Poller Runner" id="ddf.catalog.util.impl.SourcePollerRunner">
+        <AD name="Poll Interval (minutes)" id="pollIntervalMinutes" type="Integer"
             default="1"
-            description="The interval (in minutes) at which to execute successive source polling. Must be a positive, nonzero integer."/>
+            min="1"
+            description="The interval (in minutes) at which to recheck the availability of all sources. Must be at least 1 minute."/>
```

Comment:
Would it be beneficial to add a warning here about the consequences of setting the poll interval to something long?

Reply:
Seems like it might not be a bad idea to me.

Reply:
Might be helpful to add that to the documentation as well

Reply:
👍 Added documentation in the metatype description, javadoc, and documentation